client/cli/src/commands/runcmd.rs

```diff
@@ -93,6 +93,14 @@ pub struct RunCmd {
 	#[structopt(long = "unsafe-rpc-external")]
 	pub unsafe_rpc_external: bool,
 
+	/// Don't deny potentially unsafe RPCs when listening on external interfaces.
```

Comment:
` Don't deny` => `Accept`?
Double negative is hard to understand

>  Default is false. This allows exposing RPC methods publicly (same as `--unsafe-{rpc,ws}-external` )

Default allows to expose or not?

> Please do this if you know what you're doing.

If you are running a development node?